dcm2niix/dcm2niix_run.py

```diff
@@ -16,18 +16,21 @@ def convert_directory(
     output_dir,
     anonymize_bids=True,
     bids_sidecar="y",
+    comment="",
     compress_nifti="y",
     compression_level=6,
     convert_only_series="all",
     crop=False,
-    filename="%f",
+    filename="%f_%p_%t_%s",
```

Comment:
Your `manifest.json` lists the default as "%f".

Did you want consistency across these defaults?

Reply:
Here's my thinking: the scripts that perform the actions, like this one is to run the dcm2niix tool, should have the defaults set to the tool's defaults. So if you were to pip install this `dcm2niix` into another Gear, you would have all the defaults as implied by the dcm2niix tool. In this case, the dcm2niix tool defaults to `filename="%f_%p_%t_%s"`, which is why I changed it to this. But the Gear's defaults (via legacy) is `filename="%f"`. I did add more to the `README.md` to clarify this. And I am open to changing it back if you think that would be best.